### DISCUSSION TITLE
Repeating job between non-whole hours?

### DISCUSSION THREAD

**Discussion by BaconOK** [quartznet/quartznet#960] [General]
For example, this expression creates a crontrigger that repeats every 30 seconds between 9pm - 6am.  
"0/30 * 21-6 ? * *"

How can I do the same thing except I want the schedule to execute between, for example, 9:30pm - 6:30am?
Tried "0/30 * 21.5-6.5 ? * *"  or  "0/30 * 21:30-6:30 ? * *" no dice.

Thank you.

**Comment by lahma** [comment]
You should probably use a more specific trigger type. You should check out the other trigger types and also consider calendar to further limit the firing.
